proposals/Exceptions.md

```diff
@@ -724,3 +724,30 @@ follows:
 
 The event names subsection is a `name_map` which assigns names to a subset of
 the event indices (Used for both imports and module-defined).
+
+
+---
+
+## Comparisons of the two proposals
+
+- Proposal 2 has better expressiveness and possibly provides more flexibility
```

Comment:
s/has better expressiveness/is more expressive/

Reply:
Done